Formula/juju.rb

```diff
@@ -15,8 +15,11 @@ class Juju < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    system "go", "build", "github.com/juju/juju/cmd/juju"
-    system "go", "build", "github.com/juju/juju/cmd/plugins/juju-metadata"
+    git_commit = "d816abe62fbf6787974e5c4e140818ca08586e44"
```

Comment:
Can we somehow determine this hash automatically via `git`?

Reply:
@dawidd6 not without introducing a git resource, which would include the commit sha anyway. The reason we use the launchpad tar is it includes go vendored dependencies.

In our next release we will include a file that contains the git commit, at that point we can replace the hardcoded git_commit here with that file's contents.

Reply:
I'd just change the download strategy to download tagged git releases. It's way easier.